### PR TITLE
Translated shallow renderer

### PR DESCRIPTION
**Pull request by vermasachin**
Translated API Reference > Shallow Renderer.

Some issues that I'm aware of and need help on:

1. Translation for the word 'shallow'. I'm using just 'शैलो', because 'उथला' is generally used in the context of water and it's much more complicated.
2. The first title of the page 'Importing'. Import is 'आयात', but couldn't find out present continuous translation for this.
3. Couldn't figure out proper 'child, children' translations.
4. For 'assert' I have used 'जांच', couldn't find better translation.

### CODE REVIEW

File: content/docs/addons-shallow-renderer.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
 ```
 
-## Overview {#overview}
+## अवलोकन {#overview}
```
Comment (arshadkazmi42) [inline content/docs/addons-shallow-renderer.md]: `अवलोकन` => `ओवरव्यू`

You can follow #2 for these types of translations

File: content/docs/addons-shallow-renderer.md [hunk 2]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
-`shallowRenderer.render()` is similar to [`ReactDOM.render()`](/docs/react-dom.html#render) but it doesn't require DOM and only renders a single level deep. This means you can test components isolated from how their children are implemented.
+`shallowRenderer.render()` [`ReactDOM.render()`](/docs/react-dom.html#render) के समान ही है लेकिन इसे DOM की आवश्यकता नहीं है और यह केवल एक स्तर गहराई तक ही रेंडर करता है। इसका मतलब आप कौम्पोनॅन्टस का परिक्षण उनके children के लागू होने के तरीके से अलग रखकर कर सकते हैं।
 
 ### `shallowRenderer.getRenderOutput()` {#shallowrenderergetrenderoutput}
 
-After `shallowRenderer.render()` has been called, you can use `shallowRenderer.getRenderOutput()` to get the shallowly rendered output.
+`shallowRenderer.render()` के call हो जाने के बाद आप शैलो रेंडर हुए आउटपुट को पाने के लिए `shallowRenderer.getRenderOutput()` का इस्तेमाल कर सकते हैं।
 
-You can then begin to assert facts about the output.
+तब आप आउटपुट के बारे में तथ्यों की जांच कर सकते हैं।
```
Comment (arshadkazmi42) [inline content/docs/addons-shallow-renderer.md]: `जांच कर सकते हैं।` => `जांच करना सुरु कर सकते हैं। `

File: content/docs/addons-shallow-renderer.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
 ```
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+React के लिए यूनिट टेस्ट लिखते समय शैलो रेंडरिंग काफ़ी उपयोगी हो सकती है। शैलो रेंडरिंग आपको एक कौम्पोनॅन्ट को "एक स्तर गहरा" रेंडर करने देता है और आपको दृढ़ता से ये जांच करने की क्षमता प्रदान करता है कि इस कौम्पोनॅन्ट का रेंडर मेथड क्या return करेगा, child कौम्पोनॅन्टस के व्यव्हार की चिंता किये बिना, जो की instantiate या रेंडर नहीं होते। इसमें DOM की आवश्यकता नहीं होती है।
```
Comment (saranshkataria) [inline content/docs/addons-shallow-renderer.md]: दृढ़ता could be replaced by an easier to understand word?

File: content/docs/addons-shallow-renderer.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ function MyComponent() {
 }
 ```
 
-Then you can assert:
+तब आप जांच कर सकते हैं:
```
Comment (saranshkataria) [inline content/docs/addons-shallow-renderer.md]: assert has a specific meaning to it, we should just convert it to hindi

File: content/docs/addons-shallow-renderer.md [hunk 5]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+वर्तमान में शैलो टेस्टिंग की कुछ सीमाएँ हैं, नामतः ये refs के साथ काम नहीं करता।
 
-> Note:
+> टिप्पणी:
 >
-> We also recommend checking out Enzyme's [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> हम Enzyme के [शैलो रेंडरिंग API](https://airbnb.io/enzyme/docs/api/shallow.html) का भी परीक्षण करने की भी सलाह देते हैं। यह समान कार्यक्षमता पर एक अच्छा उच्च स्तरीय API प्रदान करता है।
```
Comment (saranshkataria) [inline content/docs/addons-shallow-renderer.md]: यह समान कार्यक्षमता पर एक अच्छा उच्च स्तरीय API प्रदान करता है। -> can be simplified a bit?

File: content/docs/addons-shallow-renderer.md [hunk 6]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
 
-## Reference {#reference}
+## संदर्भ {#reference}
 
 ### `shallowRenderer.render()` {#shallowrendererrender}
 
-You can think of the shallowRenderer as a "place" to render the component you're testing, and from which you can extract the component's output.
+आप जिस कौम्पोनॅन्ट का परीक्षण कर रहे हैं, shallowRenderer को आप उसे रेंडर करने के एक "स्थान" की तरह सोच सकते हैं, और जिससे आप कौम्पोनॅन्ट के आउटपुट को निकाल सकते हैं।
```
Comment (saranshkataria) [inline content/docs/addons-shallow-renderer.md]: जिससे आप कौम्पोनॅन्ट -> isse aap

File: content/docs/addons-shallow-renderer.md [hunk 7]
```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
...
-`shallowRenderer.render()` is similar to [`ReactDOM.render()`](/docs/react-dom.html#render) but it doesn't require DOM and only renders a single level deep. This means you can test components isolated from how their children are implemented.
+`shallowRenderer.render()` [`ReactDOM.render()`](/docs/react-dom.html#render) के समान ही है लेकिन इसे DOM की आवश्यकता नहीं है और यह केवल एक स्तर गहराई तक ही रेंडर करता है। इसका मतलब आप कौम्पोनॅन्टस का परिक्षण उनके children के लागू होने के तरीके से अलग रखकर कर सकते हैं।
 
 ### `shallowRenderer.getRenderOutput()` {#shallowrenderergetrenderoutput}
 
-After `shallowRenderer.render()` has been called, you can use `shallowRenderer.getRenderOutput()` to get the shallowly rendered output.
+`shallowRenderer.render()` के call हो जाने के बाद आप शैलो रेंडर हुए आउटपुट को पाने के लिए `shallowRenderer.getRenderOutput()` का इस्तेमाल कर सकते हैं।
 
-You can then begin to assert facts about the output.
+तब आप आउटपुट के बारे में तथ्यों की जांच करना शुरू कर सकते हैं।
```
Comment (saranshkataria) [inline content/docs/addons-shallow-renderer.md]: same comment about assert as above

**Comment by saranshkataria** [comment]
Great work! Added some comments. Once we are done with those, this is good to be merged :)

File: content/docs/addons-shallow-renderer.md (hunk 4 above)
Comment (vermasachin) [reply content/docs/addons-shallow-renderer.md]: What should we use? असर्ट or अस्सर्ट

Comment (saranshkataria) [reply content/docs/addons-shallow-renderer.md]: अस्सर्ट makes more sense in my opinion, thoughts @arshadkazmi42 ?
Also, whatever is decided, please do add it in #2 as well

Comment (arshadkazmi42) [reply content/docs/addons-shallow-renderer.md]: I was also thinking about this `अस्सर्ट`

**Comment by arshadkazmi42** [comment]
@vermasachin Any updates on this? Let us know if you need any help with this.

File: content/docs/addons-shallow-renderer.md (hunk 3 above)
Comment (arshadkazmi42) [reply content/docs/addons-shallow-renderer.md]: @saranshkataria I am thinking to work on this. And we will get this merged.
How about ` वास्तविकता`?

File: content/docs/addons-shallow-renderer.md (hunk 5 above)
Comment (arshadkazmi42) [reply content/docs/addons-shallow-renderer.md]: `यह उसी फंक्शनलिटी पर एक और अच्छे स्तर की API देता है` ??

File: content/docs/addons-shallow-renderer.md (hunk 3 above)
Comment (arshadkazmi42) [reply content/docs/addons-shallow-renderer.md]: `आपको दृढ़ता से ये जांच करने की क्षमता प्रदान करता है` => `और रेंडर मेथड जो रिटर्न कर रहा है उसको अस्सर्ट कर सकते हैं`

**Comment by arshadkazmi42** [comment]
@saranshkataria I have made suggested changes and added few comments. Can you review that? We will get this merged as only few fixes are present

File: content/docs/addons-shallow-renderer.md (hunk 5 above)
Comment (saranshkataria) [reply content/docs/addons-shallow-renderer.md]: sounds good

File: content/docs/addons-shallow-renderer.md (hunk 3 above)
Comment (saranshkataria) [reply content/docs/addons-shallow-renderer.md]: sounds good

**Comment by saranshkataria** [comment]
@arshadkazmi42 replied to the comments, let's get this one in once you make those changes!

**Comment by arshadkazmi42** [comment]
@saranshkataria Done with those changes. Let the tests complete, you check the changes and let's get this merged

**Pull request merged**